docs/src/optimization.md

```diff
@@ -126,3 +126,66 @@ m = Model()
 
 poly = polyhedron(m, CDDLib.Library(:exact))
 ```
+
+## Using a Polyhedra Optimizer with MathOptInterface
+
```

Comment:
Explain why one would use MOI directly instead of with JuMP (which is recommended) which is simply `Model(with_optimizer(CDDLib.Optimizer))`. The reason could be that JuMP only does `Float64` arithmetic while CDD allows rational arithmetic.